.eslintrc.json

```diff
@@ -1,22 +1,66 @@
 {
   "root": true, /* Don't read chromium's eslint config (https://eslint.org/docs/user-guide/configuring/configuration-files#cascading-and-hierarchy) */
   "extends": "standard-with-typescript",
-  "ignorePatterns": ["*.js", "*.d.ts"],
+  "ignorePatterns": [
+    "*.js",
+    "*.d.ts"
+  ],
   "env": {
     "browser": false,
     "node": true,
     "es6": true,
     "jest/globals": true
   },
-  "plugins": ["jest"],
+  "plugins": [
+    "jest",
+    "licenses"
+  ],
   "globals": {
     "chrome": "readonly"
   },
   "parserOptions": {
     "project": "./tsconfig-lint.json"
   },
   "rules": {
-    "quote-props": ["error", "consistent"],
+    "licenses/header": [
+      2,
+      {
+        "comment": {
+          "allow": "both",
+          "prefer": "line"
+        },
+        "header": [
+          "Copyright (c) {YEAR} The Brave Authors. All rights reserved.",
+          "This Source Code Form is subject to the terms of the Mozilla Public",
+          "License, v. 2.0. If a copy of the MPL was not distributed with this file,",
+          "you can obtain one at http://mozilla.org/MPL/2.0/."
+        ],
+        "altHeaders": [
+          [
+            "Copyright (c) {YEAR} The Brave Authors. All rights reserved.",
+            "* This Source Code Form is subject to the terms of the Mozilla Public",
+            "* License, v. 2.0. If a copy of the MPL was not distributed with this file,",
+            "* You can obtain one at http://mozilla.org/MPL/2.0/."
+          ],
+          {
+            "comment": {
+              "prefer": "block",
+              "allow": "both"
+            },
+            "leadingSpaces": 1,
+            "header": [
+              "This Source Code Form is subject to the terms of the Mozilla Public",
```

Comment:
This alternative (missing first line) is okay for existing code (should ideally be fixed, but out-of-scope for this PR), but not for new code. New code should have the full copyright statement the year.

Reply:
Okay, so at the moment the tool doesn't support anything like "New code must use this header" but it will generate the MPL license with the full copyright statement & year when you run `eslint --fix`. Do you think it's okay to merge as is, and maybe add that as an extension?

I think it's strictly better than what we have now (nothing).

Reply:
Yeah, I think that's fine. We can fix the other problem later :) I agree this PR is better than the current state.